tests/models/roc_bert/test_modeling_roc_bert.py

```diff
@@ -586,6 +586,24 @@ class RoCBertModelTest(ModelTesterMixin, PipelineTesterMixin, unittest.TestCase)
         else {}
     )
 
+    # TODO: Fix the failed tests when this model gets more usage
+    def is_pipeline_test_to_skip(
+        self, pipeline_test_casse_name, config_class, model_architecture, tokenizer_name, processor_name
+    ):
+        if pipeline_test_casse_name in [
+            "FillMaskPipelineTests",
+            "FeatureExtractionPipelineTests",
+            "TextClassificationPipelineTests",
+            "TokenClassificationPipelineTests",
+        ]:
+            # Get error: IndexError: index out of range in self.
+            # `word_shape_file` and `word_pronunciation_file` should be shrunk during tiny model creation,
+            # otherwise `IndexError` could occur in some embedding layers. Skip for now until this model has
+            # more usage.
+            return True
+
+        return False
```

Comment:
For some tests, we can do
```python
@unittest.skip(reason="blabla")
def test_pipeline_xxx(self):
    ...
@unittest.skip(reason="blabla")
def test_pipeline_yyy(self):
    ...
```
But for some tests, we need to check the tokenizer name or processor name, and skip the tests depending on the cases.

So having `is_pipeline_test_to_skip` is for this goal. It is also easier/more clear/concise.